src/main/io/serial.c

```diff
@@ -341,6 +341,27 @@ bool isSerialConfigValid(const serialConfig_t *serialConfigToCheck)
     for (int index = 0; index < SERIAL_PORT_COUNT; index++) {
         const serialPortConfig_t *portConfig = &serialConfigToCheck->portConfigs[index];
 
+#ifdef USE_SOFTSERIAL
```

Comment:
Should this block go before `UNUSED(serialConfigToCheck)` ???

Reply:
Actually that UNUSED statement shouldn't be there as it is used!